CorsixTH/Src/th_gfx.cpp

```diff
@@ -1332,6 +1334,7 @@ animation::animation()
   hit_test_fn = THAnimation_hit_test;
   is_multiple_frame_animation_fn = THAnimation_is_multiple_frame_animation;
   patient_effect = animation_effect::none;
+  patient_effect_offset = rand();
```

Comment:
Discussion point: should this be the system random number generator or the game random number generator (the later is intended to be reseeded where it left off for deterministic saves, though it doesn't quite work out in reality)

Reply:
I don't think there's value in having the complexity to try to keep the patient_effect_offset the same. Given that we also are not preserving game_ticks you would not reload to the exact same animation position, however as a random effect I don't think it's much of a concern. After a reload it would be hard to tell the difference between the patient reaching the end / start of a new jelly phase or the offset changing.